ideas/163-support-erc721.md

```diff
@@ -0,0 +1,78 @@
+## Preamble
+
+    Idea: 163
+    Title: Support ERC721 tokens
+    Status: Draft
+    Created: 2018-04-10
+    
+## Summary
+Add ERC721 support to Status wallet so that users can see their CryptoKitties, Punks, Celebrities, etc. alongside their other assets.
+
+## Swarm Participants
+- Lead/PM: [@rachelhamlin](https://github.com/rachelhamlin)
+- Testing & Evaluation: 
+- Contributor: [@jeluard](https://github.com/jeluard)
+- Contributor: [@goranjovic](https://github.com/goranjovic)
+- UX: [@denis-sharypin](https://github.com/denis-sharypin)
+
+## Product Overview
+
+With 100+ digital collectibles on the market and a combined sales volume greater than $50MM, it's important that we allow users to access their NFT collections inside Status right alongside their ETH and ERC20 tokens.
+
+The two primary cases we need to support are:
+1. Having my existing collectibles in my Status wallet
+2. Buying or receiving new collectibles and seeing them in my Status wallet
+
+Additional use cases may focus on gifting collectibles from one user to another, as well as sending collectibles in a chat.
```

Comment:
Also, I had an idea to add a possibility to make visible your favorite ERC721 to others by adding them to your profile